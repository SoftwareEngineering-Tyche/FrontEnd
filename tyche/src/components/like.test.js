import { shallow } from 'enzyme';
import Like from './like';


describe('renders <like />', () => {
  it('should pass shallow snapshot test', () => {
    const wrapper = shallow(<Like />);
    expect(wrapper).toMatchSnapshot();
  });
});