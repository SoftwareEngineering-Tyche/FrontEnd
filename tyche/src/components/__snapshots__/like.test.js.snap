// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders <like /> should pass shallow snapshot test 1`] = `ShallowWrapper {}`;
